sci-biology/bowtie/bowtie-2.2.4.ebuild

```diff
@@ -28,14 +30,10 @@ pkg_pretend() {
 	fi
 }
 
-src_prepare() {
-	epatch "${FILESDIR}/${P}-buildsystem.patch"
-}
-
 src_compile() {
 	emake \
 		CC="$(tc-getCC)" \
-		CPP="$(tc-getCXX)" \
+		CXX="$(tc-getCXX)" \
 		EXTRA_FLAGS="${LDFLAGS}" \
 		RELEASE_FLAGS="${CXXFLAGS} -msse2"
```

Comment:
msse2 is processor specific, so it shouldn't be added. That need to be done by the user.